dockerfiles/entrypoint.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
 set -e
+/usr/sbin/sshd
 
 if [ "$#" = 0 ]; then
 	tail -F -n0 /etc/hosts && wait
 elif [ "$1" = "controller" ]; then
-	rabbitmq-server &
+	redis-server &
 	splunk_eventgen service --role controller &
 	tail -F -n0 /etc/hosts && wait
```

Comment:
Not for this pr. Why we need to read the `hosts` here?

Reply:
etc/hosts don't mean anything, we are just tailing to keep the process running. We can tail any file whatsoever.

Reply:
maybe `tail -f /dev/null` is better.

Reply:
is there a difference between tailing /etc/hosts and /dev/null?